openerp/addons/base/res/res_partner.py

```diff
@@ -387,6 +393,18 @@ def onchange_state(self, state_id):
             return {'value': {'country_id': state.country_id.id}}
         return {'value': {}}
 
+    @api.onchange('image')
+    def onchange_email(self):
```

Comment:
It's an onchange depending on image which alters the image and is called onchange_email. I'm reasonably certain something here isn't correct.